cmake/create_test.cmake

```
@@ -87,8 +113,8 @@ endfunction()
 function(ginkgo_create_test test_name)
     ginkgo_build_test_name(${test_name} test_target_name)
     add_executable(${test_target_name} ${test_name}.cpp)
-    target_link_libraries(${test_target_name} PRIVATE ${create_test_ADDITIONAL_LIBRARIES})
-    ginkgo_set_test_target_properties(${test_target_name} ${ARGN})
+    target_link_libraries(${test_target_name})
+    ginkgo_set_test_target_properties(${test_target_name} "" ${ARGN})
```

Maybe use this 
```suggestion
    ginkgo_set_test_target_properties(${test_target_name} "_cpu" ${ARGN})
```
so that it might be clearer what the `""` parameter is for.

---

Same for the omp test